bin/BackupPC

```diff
@@ -691,14 +695,14 @@ sub Main_TryToRun_Bg_or_User_Queue
     }
     if ( $du > $Conf{DfMaxUsagePct} || $duInode > $Conf{DfMaxInodeUsagePct} ) {
         my @bgQueue = @BgQueue;
-        my $nSkip = 0;
+        my $nSkip   = 0;
 
         #
         # When the disk is too full, only run backups that will
         # do expires, not regular backups
         #
         @BgQueue = ();
-        foreach $req ( @bgQueue ) {
+        foreach $req (@bgQueue) {
```

Comment:
@shancock9 Regarding the new feature request perltidy/perltidy#26

Again we have a little bug here, since white spaces have been removed.